lib/package-task.rb

```diff
@@ -303,6 +303,8 @@ def yum_expand_variable(key)
       @rpm_version
     when "RELEASE"
       @rpm_release
+    when "PACKAGE_DIR"
+      File.basename(Dir.pwd)
```

Comment:
Sorry for this after the merge.
This is merely a question.

Is this for taking a string `fluent-package`?
Is it because we will not change this in order not to change install paths?

https://github.com/fluent/fluent-package-builder/blob/b0a2e9924cbb7f1d9600c60304baf59b630b192e/fluent-package/config.rb#L1

Reply:
Oh, will we change `PACKAGE_NAME` of `fluent-package/config.rb` in #449?
Then will we remove this `PACKAGE_DIR` in #449?

Reply:
> Is this for taking a string fluent-package?

Yes, it is equal to `fluent-package`.

> Oh, will we change PACKAGE_NAME of fluent-package/config.rb in https://github.com/fluent/fluent-package-builder/pull/449?

Yes, PACKAGE_NAME should be changed to `fluent-package` ( match to actual package name)

> Then will we remove this PACKAGE_DIR in https://github.com/fluent/fluent-package-builder/pull/449?

No. variable name will be changed, but something PACKAGE_DIR or so exists because it is intended not to change internal path (td-agent) yet. (thus it still required to represent 'td-agent' liternal somewhere.)

Reply:
Thanks! I understand the situation.

Now I commented https://github.com/fluent/fluent-package-builder/pull/449#discussion_r1178835330.
Please let us continue this talk there.